core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -110,7 +110,14 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging {
 
     def abortQueuedWorkflows(submissionId: UUID) = {
       datasource.inTransaction { dataAccess =>
-        dataAccess.workflowQuery.batchUpdateWorkflowsOfStatus(submissionId, WorkflowStatuses.Queued, WorkflowStatuses.Aborted)
+        // Update workflows in batches of 10 to limit database deadlocks between
+        // this actor and WorkflowSubmissionActor. See GAWB-2243.
+        dataAccess.workflowQuery.listWorkflowRecsForSubmissionAndStatuses(submissionId, WorkflowStatuses.Queued).flatMap { workflows =>
+          val grouped = workflows.grouped(10).map { wfs =>
```

Comment:
I'm wondering if it should go in `rawls.conf`, alongside:
```
  batchSize = 10 // used by WorkflowSubmissionActor
  processInterval = 10ms // used by WorkflowSubmissionActor
  pollInterval = 1m // used by SubmissionMonitorActor
  parallelSubmitters = 10 // used by WorkflowSubmissionActor
  maxActiveWorkflowsPerServer = 10000 // used by WorkflowSubmissionActor
```
?

Is there a rule of thumb about when to add things to rawls.conf vs reference.conf? Does rawls.conf have things that might change between environments, while reference.conf is relatively static?